core/tests/protractor_utils/editor.js

```diff
@@ -431,6 +441,10 @@ var _selectRule = function(ruleElement, interactionId, ruleName) {
     });
   });
 
+  if (skipParameters) {
```

Comment:
This bit of code should not be necessary once the function is split in two; in general radically different behaviours of a function in response to different inputs is to be avoided.